recipe/meta.yaml

```diff
@@ -74,6 +69,7 @@ about:
 extra:
   recipe-maintainers:
     - alimanfoo
+    - dstansby
```

Comment:
@dstansby have added you as a maintainer here. Please let us know whether that is ok

Reply:
👍